src/common/cockpitmemfdread.c

```diff
@@ -62,6 +62,13 @@ cockpit_memfd_read (int      fd,
       return NULL;
     }
 
+  if (buf.st_size > 100000)
+    {
+      g_set_error (error, G_FILE_ERROR, G_FILE_ERROR_INVAL,
+                   "memfd %d is unreasonably large", fd);
```

Comment:
This could print `buf.st_size`, which would help us with debugging if this ever becomes an actual problem. Like, is it only slightly too large (we got some feature creep and now actually exceed the limit), or is it something like 23984238943289 and thus clearly indicates a bug somewhere.